dbos/utils.go

```diff
@@ -0,0 +1,39 @@
+package dbos
+
+import (
+	"log/slog"
+	"net/url"
+)
+
+var (
+	logger *slog.Logger
+)
+
+// maskPassword replaces the password in a database URL with asterisks
+func maskPassword(dbURL string) string {
```

Comment:
Let's have this function return `(string, error`) and have the caller handler the error, if any